recipes/msgpack/all/conanfile.py

```diff
@@ -29,6 +29,7 @@ class MsgpackConan(ConanFile):
         "with_boost": False,
         "header_only": False
     }
+    deprecated = "msgpack-c", "msgpack-cxx"
```

Comment:
```suggestion
    deprecated = "msgpack-c or msgpack-cxx"
```
I think it has to be a string, according to https://docs.conan.io/en/latest/reference/conanfile/attributes.html#deprecated and https://github.com/conan-io/conan/blob/c1ab505b80c41567b35e8b5cbc0e00fba2e5c11d/conans/client/conanfile/configure.py#L37

Reply:
Thank you.
I misunderstood the `deprecated` attribute.